.github/CODEOWNERS

```diff
@@ -1,5 +1,30 @@
-* @DataDog/apm-ruby
+* @DataDog/ruby-guild
 
 # Public Documentation
 /docs/Compatibility.md @DataDog/documentation
-/docs/GettingStarted.md @DataDog/documentation
\ No newline at end of file
+/docs/GettingStarted.md @DataDog/documentation
+
+# Library
+lib/datadog/appsec/* @DataDog/asm-ruby
+lib/datadog/appsec.rb @DataDog/asm-ruby
+lib/datadog/tracing/* @DataDog/apm-ruby
+lib/datadog/tracing.rb @DataDog/apm-ruby
```

Comment:
I just created the https://github.com/orgs/DataDog/teams/tracing-ruby team, we can update to use this new team in this file.

Reply:
Updated.